pom.xml

```diff
@@ -0,0 +1,226 @@
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>${maven-compiler-plugin.version}</version>
+        <configuration>
+          <source>1.7</source>
+          <target>1.7</target>
+        </configuration>
+      </plugin>
+            
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>${maven-surefire-plugin.version}</version>
+        <configuration>          
+          <suiteXmlFiles>
+            <suiteXmlFile>${project.build.testOutputDirectory}/testng-unit.xml</suiteXmlFile>             
+          </suiteXmlFiles>                                       
+          <systemPropertyVariables>                                                 
+            <saucelabs.username>${saucelabs.username}</saucelabs.username>
+            <saucelabs.accessKey>${saucelabs.accessKey}</saucelabs.accessKey>                      
+            <logFilePath>${logFilePath}</logFilePath>                                                              
+          </systemPropertyVariables>          
+        </configuration>
+      </plugin>
+              
+    </plugins>      
+  </build>   
+     
+  <profiles>
+     
+    
+    <profile> 
+      <id>release</id>
+<!--      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>-->
+      <build>
+        <plugins>        
+                    
+          <!-- http://central.sonatype.org/pages/apache-maven.html-->
+          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <version>1.6.3</version>
+            <extensions>true</extensions>
+            <configuration>
+              <serverId>ossrh</serverId>
+              <nexusUrl>https://oss.sonatype.org/</nexusUrl>
+              <autoReleaseAfterClose>true</autoReleaseAfterClose>
+            </configuration>
+          </plugin>   
+          
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-source-plugin</artifactId>
+            <version>2.2.1</version>
+            <executions>
+              <execution>
+                <id>attach-sources</id>
+                <goals>
+                  <goal>jar-no-fork</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-javadoc-plugin</artifactId>
+            <version>2.9.1</version>
+            <executions>
+              <execution>
+                <id>attach-javadocs</id>
+                <goals>
+                  <goal>jar</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+          <!--      http://central.sonatype.org/pages/apache-maven.html-->      

```

Comment:
Minor but this comment is formatted differently than the one on line 166 (whitespace).